### ISSUE TITLE
Question about the batch normalization parameters (beta and gamma, not the stats)

### ISSUE THREAD

**Issue by AhmedFrikha** [cbfinn/maml#43]
Why don't you update the batch normalization parameters (beta and gamma) in the inner training loop (for j in range(num_updates - 1)) and while finetuning on a test task, like you do it for the other network parameters (weights and biases) ?

PS: I don't mean the batch normalization statistics (moving mean and variance). In your case you only use beta, because you use this function https://www.tensorflow.org/versions/r1.5/api_docs/python/tf/contrib/layers/batch_norm 

Thanks in advance!

**Comment by cbfinn** [comment]
Mostly because it was simpler to implement it without adapted the parameters. I have tried adapting the batch norm parameters as well, and didn't observe a difference on benchmarks like Omniglot and MiniImagenet.

**Comment by AhmedFrikha** [comment]
thanks
